pages/documents/MessagingChannels/MobileAppMessagingSDKforiOS/AdvancedFeatures/Logging.md

```diff
@@ -0,0 +1,114 @@
+---
+pagename: Logging
+
+Keywords:
+sitesection: Documents
+categoryname: "Messaging Channels"
+documentname: Mobile App Messaging SDK for iOS
+subfoldername: Advanced Features
+permalink: mobile-app-messaging-sdk-for-ios-advanced-features-logging.html
+indicator: messaging
+---
+
+The LivePerson Mobile SDK's Logging system has been redesigned in iOS SDK 5.2.0 to bring enhanced customer control and data safety to what we send to the system logs. This new has the following functionality located within the `LPInfraFacade`.
+
+## Precise Logging Level Control
+
+The LivePerson Mobile SDK can now be set to one of six different verbosity settings:
+
+- **TRACE** (high-traffic logs; tracks many objects as they move through the SDK)
+- **DEBUG** (details some of the inner workings of the SDK logic and network stack)
+- **INFO** (primarily large business-logic steps as the SDK connects, sends, and receives messages)
+- **WARNING** (things that went slightly wrong, but were recoverable enough that the user should not have noticed)
+- **ERROR** (things that went wrong enough that we either failed or had to resort to a fallback that the user may have noticed)
+- **OFF** (no logging output from LivePerson Mobile SDK's)
+
+As is standard with most logging systems, each of these settings also includes all levels below it. Setting the SDK Logging Level to **OFF** disables SDK logging entirely.  The LivePerson Mobile SDK's will retain up to 100 lines of logs for combined all logging levels regardless of the logging level.  This means that if the logging level is set to "ERROR" for example, you may not be able to retrieve the last 100 logs recorded.    
+
+The default log level is **INFO** in Release builds, and **TRACE** in Debug builds.
```

Comment:
I am wondering if the default should indeed be INFO?
Any reason why this is not ERROR? For a bran which is heavy use of Mobile, this INFO might fill up lot of their storage. Is that what they want?

Reply:
@randymclainLP